axlearn/audio/asr_decoder_test.py

```diff
@@ -89,6 +91,75 @@ def test_map_label_sequences(
         )
 
 
+class ValidCtcSeqTest(TestCase):
+    def get_logits_and_labels(self, batchsize, timesteps, labelsteps, nclasses):
```

Comment:
Same comment as above re types and naming -- also, should we use `jnp`/`jax.random` here and elsewhere? Besides consistency, we can avoid `np.random` calls that may depend on a global seed?